libs/blocks/chart/chart.js

```diff
@@ -263,8 +288,12 @@ const init = async (el) => {
   const data = await fetchData(dataLink);
   if (!data) return;
 
-  // ToDo: Replace colors MWPW-112994
-  const colors = ['red', 'blue', 'green'];
+  const authoredColor = Array.from(chartStyles)?.find(style => style in colorPalette);
+  const colors = getColors(authoredColor);
+  const useContainerStyles = Array.from(chartStyles)?.indexOf(CONTAINER_STYLES);
+  if (useContainerStyles !== -1) {
+    container.classList.add(CONTAINER_STYLES_CLASSNAME);
+  }
```

Comment:
Newline after variable block
```suggestion
  const useContainerStyles = Array.from(chartStyles)?.indexOf(CONTAINER_STYLES);

  if (useContainerStyles !== -1) {
    container.classList.add(CONTAINER_STYLES_CLASSNAME);
  }
```

Reply:
Updated.